pkgs/by-name/fr/frequest/package.nix

```diff
@@ -33,6 +33,15 @@ stdenv.mkDerivation (finalAttrs: {
   ];
   sourceRoot = "frequest";
 
+  patches = [
+    # TODO: Remove once a version a new version is delivered.
```

Comment:
This is not phrased well. I also wouldn't use the `TODO` prefix. However it's still understandable what is the purpose of the comment, so I'll merge this if CI will be green before you'll push a fix.

Reply:
Should be better now.